man/whatNWISdata.Rd

```diff
@@ -0,0 +1,85 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
So at some point we got an whatNWISdata and whatNWISData (capital D) in here. I'll fix it, but that's why this seems to be a totally new file...because it is.